test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -243,6 +335,40 @@ describe('CSOT spec prose tests', function () {
        *   - Expect this to fail with a timeout error.
        * 1. Verify that a `find` command was executed against the `keyvault.datakeys` collection as part of the `decrypt` call.
        */
+      it('times out due to timeoutMS', async function () {
+        const datakeyId = await clientEncryption.createDataKey('local');
+        expect(datakeyId).to.be.instanceOf(Binary);
+        expect(datakeyId.sub_type).to.equal(Binary.SUBTYPE_UUID);
+
+        const encrypted = Binary.createFromBase64(
+          'Af6ie/LRP0uoisAZthHPUs0CKzTBFIkJr8kxmOk1pV1C/6K54otT8QvNJgNTNG2CNpThhfdXaObuOMMReNlTgwapqPYCb/HJRQ1Nfma6uA3cTg==',
+          6
+        );
+        expect(encrypted).to.be.instanceOf(Binary);
+        expect(encrypted.sub_type).to.equal(Binary.SUBTYPE_ENCRYPTED);
+
+        await internalClient
+          .db()
+          .admin()
+          .command({
+            configureFailPoint: 'failCommand',
+            mode: {
+              times: 1
+            },
+            data: {
+              failCommands: ['find'],
+              blockConnection: true,
+              blockTimeMS: 150
+            }
+          } as FailPoint);
+
+        const commandStarted: CommandStartedEvent[] = [];
+        keyVaultClient.on('commandStarted', ev => commandStarted.push(ev));
+
+        const err = await clientEncryption.decrypt(encrypted).catch(e => e);
+        expect(err).to.be.instanceOf(MongoOperationTimeoutError);
+        expect(commandStarted[0]).to.containSubset({ commandName: 'find' });
```

Comment:
Same as other comments regarding checking the command was executed against the correct namespace